.github/workflows/update_dependencies.yml

```
@@ -0,0 +1,38 @@
+name: Update dependencies
+
+on:
+    workflow_dispatch:
+
+jobs:
+    update_dependencies:
+        runs-on: ubuntu-latest
+        steps:
+            - name: "Checkout"
+              uses: actions/checkout@v3
+              with:
+                  ref: main
+                  fetch-depth: 0
+            - name: Set up Python 3.8
+              id: py
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.8
+                  cache: pip
+                  cache-dependency-path: "requirements/*.txt"
+            - name: Update dependencies
+              run: |
+                  pip install pip-tools
+                  make update-deps
+            - name: Commit changes and open PR
+              run: |
+                  export WEEK="w$(date +%V)"
```

```suggestion
                  export WEEK="W$(date +%V)"
```
I will die on [this hill](https://en.wikipedia.org/wiki/ISO_week_date).